adafruit_ags02ma.py

```diff
@@ -80,30 +118,40 @@ def __init__(self, i2c_bus, address=AGS02MA_I2CADDR_DEFAULT):
         except RuntimeError as exc:  # a CRC error or something!
             raise RuntimeError("Failed to find AGS02MA - check your wiring!") from exc
 
-    def firmware_version(self):
+    def firmware_version(self) -> int:
         """Return 24-bit value which contains the firmware version"""
         return self._read_reg(_AGS02MA_VERSION_REG, 30)
 
     @property
-    def gas_resistance(self):
+    def gas_resistance(self) -> float:
         """The resistance of the MEMS gas sensor"""
         return self._read_reg(_AGS02MA_GASRES_REG, 1500) * 100  # in 0.1Kohm
 
     @property
-    def TVOC(self):  # pylint: disable=invalid-name
-        """The calculated Total Volatile Organic Compound measurement, in ppb"""
+    def TVOC(self) -> int:  # pylint: disable=invalid-name
+        """The calculated Total Volatile Organic Compound measurement, in ppb
+
+        :raises RunTimeError: When the sensor still preheating
+        """
         val = self._read_reg(_AGS02MA_TVOCSTAT_REG, 1500)
         status = val >> 24
-        # print(hex(status))
+
         if status & 0x1:
             raise RuntimeError("Sensor still preheating")
         return val & 0xFFFFFF
 
-    def set_address(self, new_addr):
+    def set_address(self, new_addr: int) -> NoReturn:
```

Comment:
I've not seen `NoReturn` before, is that different from using `None` as the return type?

Reply:
My bad. NoReturn should be use whe the function indeed has no return, ex: exit functions and other that would exit the program. https://docs.python.org/3/library/typing.html#typing.NoReturn
Corrected pls see https://github.com/adafruit/Adafruit_CircuitPython_AGS02MA/pull/3/commits/7d2eb7a3242cd3fcb3af70b409ba43b5e1fec6b6